db/pf-schema-X.Y.Z.sql

```diff
@@ -1107,6 +1107,78 @@ CREATE TABLE radius_audit_log (
 ) ENGINE=InnoDB;
 
 --
+-- Table structure for table `node_option82`
+--
+
+CREATE TABLE `node_option82` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `created_at` TIMESTAMP NOT NULL,
+  `mac` varchar(17) NOT NULL,
+  `option82_switch` varchar(17) NULL,
+  `switch_id` varchar(17) NULL,
+  `port` varchar(8) NOT NULL default '',
+  `vlan` varchar(255) default NULL,
+  `circuit_id_string` varchar(255) default NULL,
+  `module` varchar(255) default NULL,
+  `host` varchar(255) default NULL,
+  UNIQUE KEY mac (mac)
+) ENGINE=InnoDB;
+
+--
+-- Table structure for table `node_option82_history`
+--
+
+CREATE TABLE `node_option82_history` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
This is the exact same logic as for the iplog and iplog_history tables.
I don't see the point of having an "id" if we are "not using it"...